### ISSUE TITLE
Have a progress bar for S3 upload

### ISSUE THREAD

**Issue by moulip** [wallix/awless#78]
When issuing `awless create storageobject`, it would be nice to have a progress bar.
Thanks dev-team ;-)

**Comment by simcap** [comment]
Done. 

This is showing progress dynamically using correct bytes unit (Mega, Giga, ...):

```sh
awless create storageobject bucket=wallix-simon-test file=./awless

Confirm? (y/n): y
[info]    uploading './awless'

21.3 MB/36.8 MB
```
(Note this it not an actual progress bar as per say yet for implementation reason. See commit if interested)

**Comment by simcap** [comment]
Available in master or in upcoming 0.0.21
